### PR TITLE
MGMT-18475: disabling CGO

### PR DESCRIPTION
**Pull request by eifrach**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,9 +10,7 @@ BIN = $(ROOT_DIR)/build
 # Multiarch support.  Transform argument passed from docker buidx tool to go build arguments to support cross compiling
 GO_BUILD_ARCHITECTURE_VARS := $(if ${TARGETPLATFORM},$(shell echo ${TARGETPLATFORM} | awk -F / '{printf("GOOS=%s GOARCH=%s", $$1, $$2)}'),)
 CGO_FLAG := $(or $(CGO_FLAG),0)
```
Comment (danmanor) [inline Makefile:12]: no need for this right

File: Makefile [hunk 2]
```diff
@@ -10,9 +10,7 @@ BIN = $(ROOT_DIR)/build
...
-ifeq ($(TARGETPLATFORM),linux/amd64)
-	CGO_FLAG=1
-endif
+
 GO_BUILD_VARS := CGO_ENABLED=$(CGO_FLAG) $(GO_BUILD_ARCHITECTURE_VARS)
```
Comment (danmanor) [inline Makefile:14]: No need for `CGO_ENABLED=$(CGO_FLAG)` also

Comment (eifrach) [reply Makefile:14]: the default is `CGO_ENABLED=1` so you must disable it

File: Makefile (hunk 1 above)
Comment (eifrach) [reply Makefile:12]: the default is CGO_ENABLED=1 so you must disable it

File: Makefile (hunk 2 above)
Comment (danmanor) [reply Makefile:14]: so can we just `CGO_ENABLED=0` ?

**Pull request merged**